### PR TITLE
Update cjapy.py

### PR DESCRIPTION
**Pull request by admoseremic**
Added getPersonProfiles function - an easy way to grab person level profiles from CJA for ML training purposes.

### CODE REVIEW

File: cjapy/cjapy.py [hunk 1]
```diff
@@ -2560,3 +2560,256 @@ def clean_column_name(col):
...
+                        "col1": {
+                            "func": "cumul",
+                            "description": "Cumulative",
+                            "n": 0,
+                            "col": 1
+                        },
+                        "col2": sampleSeed
+                        }
+                    },
+                    "col2": 1664525
```
Comment (pitchmuc) [inline cjapy/cjapy.py:2660]: Are we sure about having fixed value ? 
I am always thinking that hardcoded value may be a typo so just asking.

File: cjapy/cjapy.py [hunk 2]
```diff
@@ -2560,3 +2560,256 @@ def clean_column_name(col):
...
+                    },
+                    "col2": 1013904223
```
Comment (pitchmuc) [inline cjapy/cjapy.py:2662]: Same here with hardcoded value.

File: cjapy/cjapy.py [hunk 3]
```diff
@@ -2560,3 +2560,256 @@ def clean_column_name(col):
         
         df.columns = [clean_column_name(col) for col in df.columns]
         return df
+
+    def getPersonProfiles(
+        self,
+        dataviewId: str = None,
+        personId: str = "variables/adobe_identitynamespace_personid",
+        featureMetrics: List[str] = None,
```
Comment (pitchmuc) [inline cjapy/cjapy.py:2568]: @admoseremic 
FYI : The typing such as `List[str]` is only supported above a certain version of python, so I usually remove them before going live so the module can be used with python 3.6 and above (when it was not available). 
So you know why you may not see them.

File: cjapy/cjapy.py (hunk 1 above)
Comment (admoseremic) [reply cjapy/cjapy.py:2660]: https://en.wikipedia.org/wiki/Linear_congruential_generator

The hard-coded values are common values used in the LCG random number generator. These values are used to create a random sample of users from CJA to pull from.

**Pull request merged**